application/services/bi.py

```diff
@@ -10,3 +13,16 @@ def host(self) -> str:
 
     def port(self) -> int:
         return 80
+
+    def url_env_var(self) -> str:
+     return "BI_SERVICE_URL"
+
+    def atualizarDashTwitter(self, ids: dict):
```

Comment:
Would be good have a snake case method name (and also in english), following [PEP 8 recommendations](https://peps.python.org/pep-0008/#method-names-and-instance-variables).

I can suggest an update:

```suggestion
    def retrieve_twitter_dashboard_data(self, ids: dict):
```

Reply:
Also, would be good to have a type for `ids` parameter and the method return, e.g.:

```suggestion
    def retrieve_twitter_dashboard_data(self, ids: IDs) -> TwitterDashboardData:
    a
```

A good way to create these types (we can call also **models**) is using `pydantic` library, installed automatically with `FastAPI`. So you can just do something like that on `application/models/bi.py`:

```python
from pydantic import BaseModel

class IDs(BaseModel):
    user_id: str
    twitter_user_id: str

class TwitterDashboardData(BaseModel):
    user_id: str
    graph_likes: bytes
   graph_likes_average: bytes
```

and then import on your other files as and enjoy it:

```python
from application.models.bi import IDs
from application.models.bi import TwitterDashboardData
```